jenkins/security/whitesource_container_scan.groovy

```diff
@@ -0,0 +1,202 @@
+pipeline {
+    agent {
+        node {
+            label 'docker-k8-deployment-node'
+        }
+    }
+    
+    options {
+        timeout(time: 240, unit: 'MINUTES')
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '30', numToKeepStr: '30'))
+        ansiColor('xterm')
+    }
+
+    environment {
+        PODS_ON_PRIMARY = "true"
```

Comment:
Do we need to allow POD creation on the Primary node? I guess we are planning to create a 1+3 setup. In that case this should be `false` 
@swanand-gadre - Could you please confirm.

Reply:
removed ${PODS_ON_PRIMARY}

Reply:
it's still there https://github.com/Abhirup-A-G/cortx-re/blob/CORTX-31409/jenkins/security/whitesource_container_scan.groovy#L16